backend/src/main/java/de/neuefische/cgnjava222/zoomanagement/zoo/plant/PlantController.java

```diff
@@ -18,5 +18,18 @@ public PlantController(PlantService plantService) {
 
     @GetMapping()
     List<Plant> getPlants() {
-        return plantService.getAllPlants();}
+        return plantService.getAllPlants();
+    }
+
+    @PostMapping(path = "")
+    public ResponseEntity<Plant> addPlant(
+            @RequestBody PlantWithOutId plantName) {
```

Comment:
or maybe `NewPlant`?

```suggestion
            @RequestBody PlantWithoutId plantName) {
```